### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my component!

### CODE REVIEW

File: Components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.navigation-box{
+    border: 1px solid var(--gray);
+    margin: auto;
+    /* width:80%; */
```
Comment (vermasavita) [inline Components/Navigation/navigation.css]: You can delete this comment

File: Components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	justify-content: space-between;
+	align-items: center;
+}
+
+.nav-bar-logo {
+	font-family: 'Gill Sans MT', Calibri, ;
+	font-size: 2.5rem;
+	color: var(--blue);
+}
+
```
Comment (vermasavita) [inline Components/Navigation/navigation.css:45]: Remove this space

File: Components/Navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.link-text{
+    font-size: 1.5rem;
+}
+.link-text:hover{
+    color: var(--text-color);
+}
+
+.navigation-container {
+    width: 100%;
+    height: 50px;
```
Comment (vermasavita) [inline Components/Navigation/navigation.css]: You can use rem instead for responsivness.

File: Components/Navigation/navigation.html [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+            <li class="nav-link-item">
+              <a href="../Button/button.html">Button</a>
+            </li>
+            <li class="nav-link-item"><a href="#Card">Card</a></li>
+            <li class="nav-link-item"><a href="#Input">Input</a></li>
+            <li class="nav-link-item">
+              <a href="../Responsive Images/image.html">Images</a>
+            </li>
+            <!-- <li class="nav-link-item"><a href="#Lists">Lists</a></li>
+                <li class="nav-link-item"><a href="#Modal">Modal</a></li>
```
Comment (vermasavita) [inline Components/Navigation/navigation.html]: delete these comment if no use

**Review by vermasavita — APPROVED** [review]

File: Components/Navigation/navigation.css [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  padding: 6px 10px;
+}
+
+form input {
+  flex-grow: 1;
+  border-radius: 50px;
+  outline: none;
+  font-size: 1.1rem;
+}
+
```
Comment (anupkgurung) [inline Components/Navigation/navigation.css]: element specific css might create an issue if you'll have this css file as a commonly used one.  I am assuming it is nav related so it is common file for multiple pages and if you would have form input or button than these style would also get applied to them unless you want them too.

**Review by anupkgurung — APPROVED** [review]
left a suggestion. Try to avoid element specific css.

**Pull request merged**